cmp/grid/columns/Column.js

```diff
@@ -258,6 +260,15 @@ export class Column {
         this.setValueFn = withDefault(setValueFn, this.defaultSetValueFn);
         this.getValueFn = withDefault(getValueFn, this.defaultGetValueFn);
 
+        this.autoSizeOptions = {
+            enabled: withDefault(this.resizable && !this.elementRenderer, true),
+            sampleCount: 10,
+            buffer: 5,
+            minWidth: this.minWidth,
+            maxWidth: this.maxWidth,
```

Comment:
I like that these can be controlled for autoSize, separately from the limits imposed on the column for manual resizing. 👍